frontend/public/components/overview/pods-overview.tsx

```diff
@@ -70,14 +72,21 @@ const isPodWithoutImageId = (pod: PodKind) =>
   pod.status?.phase === 'Pending' &&
   pod.status?.containerStatuses?.some((containerStatus) => !containerStatus.imageID);
 
-const podCompare = (pod1: PodKind, pod2: PodKind): number => {
+export const podCompare = (pod1: PodKind, pod2: PodKind): number => {
   const error1 = isPodError(pod1);
   const error2 = isPodError(pod2);
 
   if (error1 !== error2) {
     return error1 ? 1 : 0;
   }
 
+  const evicted1 = isEvicted(pod1);
+  const evicted2 = isEvicted(pod2);
+
+  if (evicted1 !== evicted2) {
+    return evicted1 ? 1 : -1;
+  }
```

Comment:
That's better then the error compare. :+1:

So we should use this above as well (with opposite order, which means `? -1 : 1`)